public/index.css

```diff
@@ -0,0 +1,658 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700;300;400;500&display=swap');
+
+* {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  list-style: none;
+}
+
+body {
+  background: #f6f6f6;
+  margin: 0;
+}
+
+.header {
+  background-image: url('../assets/images/header_background.png');
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+}
+
+@media (max-width: 782px) {
+  .header {
+    background-color: rgba(220, 220, 220, 0.3);
+  }
```

Comment:
I noticed you used both max-width & min-width to define your media query.
A mobile first approach is required in this project. Kindly remove the max-width definition.

If there is a need clarification, kindly reachout.

### :slightly_smiling_face: